test/helpers/HoldRequestDispatcher.test.js

```diff
@@ -10,16 +10,17 @@ const SCSBApiHelper = require('../../src/helpers/SCSBApiHelper')
 
 const onSiteEddRequest = require('../fixtures/onSiteEddRequest')
 const onSiteRequest = require('../fixtures/onSiteRequest')
-const recapRequest = require('../fixtures/nyplRecapRequest')
+const recapNyplItemRequest = require('../fixtures/nyplRecapRequest')
+const partnerItemRequest = require('../fixtures/partnerItemRequest')
 
-const mixedRecords = [ onSiteEddRequest, recapRequest ]
+const mixedRecords = [ onSiteEddRequest, recapNyplItemRequest, partnerItemRequest ]
 
 describe('HoldRequestConsumer Lambda: HoldRequestDispatcher', () => {
   describe('sortRecords', () => {
-    it('should sort records by holding location', () => {
+    it('should sort records according to nyplSource and holding location', () => {
       const sortedRecords = HoldRequestDispatcher.sortRecords(mixedRecords)
-      expect(sortedRecords.scsb.length).to.equal(1);
-      expect(sortedRecords.onSite.length).to.equal(1);
+      expect(sortedRecords.scsb.length).to.equal(2)
+      expect(sortedRecords.onSite.length).to.equal(1)
```

Comment:
If easy, could you verify membership? Minimally, just confirm that `sortedRecords.onSite[0]` is the one you expect?

Reply:
Good idea. Added that both checks are true.